packages/pn-commons/src/components/Footer/__test__/Footer.test.tsx

```diff
@@ -63,13 +64,19 @@ describe('Footer Component', () => {
     }
     languageKeys.forEach((currentDropdownLanguage, index) => {
       fireEvent.click(dropdownLanguageButton);
+      sessionStorage.setItem('lang', currentDropdownLanguage);
+
       const languageSelector = screen.getByRole('presentation');
       expect(languageSelector).toBeInTheDocument();
       const languageOptions = languageSelector?.querySelectorAll('ul li');
       expect(languageOptions).toHaveLength(Object.keys(LANGUAGES[currentDropdownLanguage]!).length);
       const languageOptionsArray = Array.from(languageOptions);
       expect(languageOptionsArray[index]).toHaveTextContent(expectedLanguagesLabels[index]);
       fireEvent.click(languageOptionsArray[index]);
```

Comment:
move the click after the forEach and choose a language that isn't the first. after click check that the language is selected